client/src/components/filmBox.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FilmItem from './filmItem';
+
+export default function FilmBox({ films }) {
+  return (
+    <div className='filmBox'>
+      {films.results.map((e) => (
```

Comment:
What does the name `e` stand for? I guess you meant `element`, although some people my mistakenly assume you meant `event`. If you named it `film` there would be no need to guess.

Reply:
you are right. this is completely because of rush of last night. I changed it to 'film'